industrial_ci/src/tests/source_tests.sh

```diff
@@ -97,6 +97,29 @@ function run_clang_tidy_check {
     fi
 }
 
+function run_pylint_check {
+    local target_ws=$1
+    local status=0
+    local exit_code=0
+
+    local -a pylint_versions
+    ici_parse_env_array pylint_versions PYLINT_VERSIONS
+    local -a pylint_args
+    ici_parse_env_array pylint_args PYLINT_ARGS
+
+    for cmd in "${pylint_versions[@]}"; do
+        ici_run "install_$cmd" ici_install_pkgs_for_command "$cmd" "$cmd"
+        set +e
+        ici_with_ws "$target_ws" ici_run "run_$cmd" ici_exec_in_workspace "$extend" "$target_ws" "$cmd" "${pylint_args[@]}" "$(find "$target_ws/src" -iname "*.py")"; status=$?
+        set -e
```

Comment:
this is not yet doing the right thing
I wanted to save the exit code of each `pylint_version` to `status` and only exit with exit_code `1` at the end if one of them returned with exit code !=0

how can I get can I access the exit code of `cmd`?

Reply:
Try something like https://github.com/ros-industrial/industrial_ci/blob/master/industrial_ci/src/tests/clang_format_check.sh#L36